doc/source/user_guide/basics.rst

```diff
@@ -1179,9 +1179,11 @@ a single value and returning a single value. For example:
 
    df4
 
+
    def f(x):
        return len(str(x))
 
+
```

Comment:
I _believe_ this is what's causing the CI to fail. With nbsphinx, multiple blank lines will generate a warning causing our docs build to fail (docs are still built successfully though). Not sure if there is a good way to handle this.